persistence/trees/trees_test.go

```diff
@@ -16,3 +23,28 @@ func TestTreeStore_DebugClearAll(t *testing.T) {
 	// TODO: Write test case for the DebugClearAll method
 	t.Skip("TODO: Write test case for DebugClearAll method")
 }
+
+func TestStateTree_Operations(t *testing.T) {
```

Comment:
Replace `Operations` with `Basic` or `General` so it's clearer that you just added a single catch all unit test